app/main.py

```diff
@@ -1,6 +1,23 @@
+import os
+import requests
+
+API_KEY = os.environ.get("API_KEY")
+CITY = "Paris"
+URL = "http://api.weatherapi.com/v1/current.json"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    print(f"Performing request to Weather API for city {CITY}...")
+    response = requests.get(URL + f"?key={API_KEY}&q={CITY}")
```

Comment:
I know it's not in the checklist, but it's better to create `params` variable with key and q and then do:
```python
requests.get(URL, params=params)
```